packages/donations-app/app/components/checkout/checkoutController.js

```diff
@@ -1,69 +1,34 @@
 angular.module('aliceApp')
-  .controller('CheckoutController', ['$rootScope', '$uibModal', '$http', '$stateParams', '$state', '$sce', '$timeout', 'AuthService', 'ProjectService', 'NotificationService', 'API', '$scope', 'MANGO', function ($rootScope, $uibModal, $http, $stateParams, $state, $sce, $timeout, AuthService, ProjectService, NotificationService, API, $scope, MANGO) {
+  .controller('CheckoutController', ['$rootScope', '$uibModal', '$http', '$stateParams', '$state', '$sce', '$timeout', 'AuthService', 'ProjectService', 'NotificationService', 'API', '$scope', 'MANGO', 'CheckoutService', function ($rootScope, $uibModal, $http, $stateParams, $state, $sce, $timeout, AuthService, ProjectService, NotificationService, API, $scope, MANGO, CheckoutService) {
 
-    // Set MANGOPAY API base URL and Client ID
-    mangoPay.cardRegistration.baseURL = MANGO.url;
-    mangoPay.cardRegistration.clientId = MANGO.clientId;
-
-    var amounts = {
+    const amounts = {
```

Comment:
Why do we need to have this (amounts) object? I think it would be cleaner to have a list of selectable amounts and use it to generate amount selector

Reply:
It'll be refactored in the new selector.